src/schema/sources.ts

```diff
@@ -1594,7 +1633,10 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       ctx: Context,
       info,
     ): Promise<Connection<GQLSource>> => {
-      const filter: FindOptionsWhere<Source> = { active: true };
+      const filter: FindOptionsWhere<Source> = {
+        active: true,
+        type: Not(SourceType.User),
```

Comment:
should this be applied to stuff like trendingSource, popularSource, topVideoSource and so on?

Also mutations like leaveSource and deleteSource, not sure if they would pass and let user delete/leave?

Reply:
Wasn't supposed to commit this part 🙈
Going to do that in a separate PR